test/utils.spec.js

```diff
@@ -110,7 +110,7 @@ describe('utils', () => {
 
   describe('fromPublicKeyKey', () => {
     it('round trips', (done) => {
-      makePeers(50, (err, peers) => {
+      makePeers(10, (err, peers) => {
```

Comment:
Why reduce the number of peers created?

Reply:
timeout issues on Travis CI

Reply:
Let's update the timeout for now. We will come back at speed later.